src/Utilities/DOM.jl

```diff
@@ -292,6 +292,9 @@ function escapehtml(text::AbstractString)
             char === '"'  ? write(buffer, "&quot;") : write(buffer, char)
         end
         String(take!(buffer))
+    end
+    if occursin("\n", text)
+        replace(text, "\n"=>"&#10;")
```

Comment:
This should go into the `if`/`for` above. Currently it's actually breaking the escaping (the top `if` never returns from the function).

Reply:
Yeah, I realised that. The problem here though is that if I replace it there, the code block doesn't render properly. I believe highlight.js needs those "/n"

Reply:
Right. Should be okay if we only escape `\n`-s in attributes? I think we could implement that by adding a keyword argument to `escapehtml` and toggling that wherever we're escaping the attribute contents?